pkg/skaffold/runner/deployer.go

```diff
@@ -59,20 +62,21 @@ func GetDeployer(runCtx *v2.RunContext, provider deploy.ComponentProvider, label
 	if runCtx.Opts.Apply {
 		return getDefaultDeployer(runCtx, provider, labels)
 	}
-
 	deployerCfg := runCtx.Deployers()
 
 	var deployers []deploy.Deployer
+
 	for _, d := range deployerCfg {
 		dCtx := &deployerCtx{runCtx, d}
+		// TODO: deprecate HelmDeploy
 		if d.HelmDeploy != nil {
 			h, err := helm.NewDeployer(dCtx, labels, provider, d.HelmDeploy)
 			if err != nil {
 				return nil, err
 			}
 			deployers = append(deployers, h)
 		}
-
+		// TODO: deprecate KptDeploy
```

Comment:
```suggestion
		// TODO(yuwenma)[07/23/21]: deprecate KptDeploy
```

Reply:
ack. This TODO is removed in the next PR which is diff-based from this one. I'll leave the TODO untouched so as there's one less merge-conflict to fix 😄